member/resource.py

```diff
@@ -21,5 +21,23 @@ class Member(MethodResource):
     def get(self, id):
         return Model.query.get_or_404(id)
 
+    @doc(
+        summary="Modify a particular member",
+        description="""Modifies a member record with a given id with
+        the data in the request body."""
+    )
+    @auth_required
+    @use_kwargs(Schema)
+    @marshal_with(Schema)
+    def put(self, id, **att_data):
+        att = Model.query.get_or_404(id)
+
+        for field in att_data:
+            setattr(att, field, att_data[field])
+
+        db.session.add(att)
```

Comment:
You don't need to add existing instances to the session again. Just as long as you commit at the end things will work :)

Reply:
I don't actually know anything about Flask 😛 but I shall change